src/openforms/contrib/haal_centraal/tests/test_integration.py

```diff
@@ -0,0 +1,147 @@
+from pathlib import Path
+from unittest.mock import MagicMock, patch
+
+from django.test import TestCase
+
+from openforms.authentication.constants import AuthAttribute
+from openforms.authentication.utils import store_auth_details, store_registrator_details
+from openforms.config.models import GlobalConfiguration
+from openforms.forms.tests.factories import FormStepFactory
+from openforms.prefill import prefill_variables
+from openforms.submissions.tests.factories import SubmissionStepFactory
+from openforms.typing import JSONValue
+from openforms.utils.tests.vcr import OFVCRMixin
+
+from ..models import HaalCentraalRequestOptions
+from .factories import HaalCentraalConfigFactory
+
+CONFIGURATION = {
+    "display": "form",
+    "components": [
+        {
+            "key": "bsn",
+            "type": "textfield",
+            "label": "BSN",
+            "prefill": {
+                "plugin": "haalcentraal",
+                "attribute": "burgerservicenummer",
+            },
+            "multiple": False,
+        },
+        {
+            "key": "address",
+            "type": "textfield",
+            "label": "Address",
+            "prefill": {
+                "plugin": "haalcentraal",
```

Comment:
Can you please use the [constant](https://github.com/open-formulieren/open-forms/blob/master/src/openforms/prefill/contrib/haalcentraal_brp/plugin.py#L27) instead? We need to rename the plugin identifier at some point, this is one of the last places where the incorrect assumption is made that `haalcentraal == brp personen`